net/nimble/host/src/ble_sm.c

```diff
@@ -892,6 +895,72 @@ ble_sm_key_dist(struct ble_sm_proc *proc,
     }
 }
 
+static int
+ble_sm_chk_store_overflow_once(int is_our_sec, uint16_t conn_handle)
```

Comment:
OK, I can do that.  Where are the other places where such a helper could be used?  This is the only place I can see.